bindings/capacitorjs/ios/Plugin.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,619 @@
+// !$*UTF8*$!
```

Comment:
Do we need to add `libc++` under `Link Binary with Libraries`?

<img width="859" alt="image" src="https://user-images.githubusercontent.com/19519564/215919084-1d4d2936-e592-402b-950f-fa53dbb516cb.png">


Without adding that I get this error mentioning a lot of C++ called from RocksDB:

```
ld: warning: Could not find or use auto-linked library 'WalletFramework'
Undefined symbols for architecture arm64:
  "std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::at(unsigned long) const", referenced from:
      rocksdb::Customizable::GetOptionName(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) const in WalletFramework(customizable.o)
  "std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::find(char, unsigned long) const", referenced from:
      rocksdb::(anonymous namespace)::BackupEngineImpl::FileInfo::GetDbFileName() in WalletFramework(backup_engine.o)
      rocksdb::(anonymous namespace)::BackupEngineImpl::BackupMeta::LoadFromFile(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::unordered_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, unsigned long long, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const, unsigned long long> > > const&, rocksdb::RateLimiter*, rocksdb::Logger*, std::__1::unordered_set<std::__1::basic_string<char, 
...
```

Reply:
I wasn't able to reproduce it, maybe my XCode version is a bit old, but added it same as it fix your errors

Reply:
I might need to update mine tbh. I'm still on 14.1

Reply:
added again thanks!